lib/Doctrine/DBAL/Driver/Exception/UnknownFetchMode.php

```diff
@@ -9,10 +9,7 @@
 
 final class UnknownFetchMode extends DBALException
 {
-    /**
-     * @param mixed $fetchMode
-     */
-    public static function new($fetchMode) : self
+    public static function new(int $fetchMode) : self
```

Comment:
The only place where it's being used is:

https://github.com/doctrine/dbal/blob/97e0c2bb7386288ef6954aeb10f990b9231cb517/lib/Doctrine/DBAL/Driver/PDOStatement.php#L208-L215